crypto/asn1/a_gentm.c

```diff
@@ -157,10 +157,10 @@ int ASN1_GENERALIZEDTIME_print(BIO *bp, const ASN1_GENERALIZEDTIME *tm)
     if (BIO_printf(bp, "%s %2d %02d:%02d:%02d%.*s %d%s",
                    _asn1_mon[M - 1], d, h, m, s, f_len, f, y,
```

Comment:
I'm directly opposed to having to separate source copies of the same array.  Someone is bound to forget updating both consistently, and the long term consequences are usually hellish.

Better, then, to have it in a header file, like @InfoHunter says.

However, this is only libcrypto internal anyway, right?  Not sure if it makes a huge difference to have it defined static in that case.  I *do* think it's valuable to constify it, though.